src/components/popover/popover.tsx

```diff
@@ -659,6 +663,7 @@ export class EuiPopover extends Component<Props, State> {
       display,
       onTrapDeactivation,
       buffer,
+      ariaLabelledId,
```

Comment:
Then, to deconstruct a prop with a hyphen in it, we'll need to rename in to a valid JS variable
```suggestion
      'aria-label': ariaLabel,
      'aria-labelledby': ariaLabelledBy,
```